### PR TITLE
[WIP] Update issue templates

### PR DESCRIPTION
**Pull request by prasanthpul**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
```
Comment (askhade) [inline .github/ISSUE_TEMPLATE/bug_report.md:15]: Change the steps to something like this:
- Describe steps/code to reproduce the behavior.
- Attach the ONNX model to the issue (where applicable) to expedite investigation.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
+ - Version [e.g. 22]
+
+**Smartphone (please complete the following information):**
```
Comment (askhade) [inline .github/ISSUE_TEMPLATE/bug_report.md:31]: Merge Desktop and Smartphone sections to System Information
Under this 
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):
- ONNX installed from (source or binary):
- ONNX version:
- Python version:

**Comment by askhade** [comment]
In the same PR add templates for Feature Request and Question?

**Comment by prasanthpul** [comment]
this was just a test PR

**Pull request closed without merging**